File: FrontEnd/src/app/features/invoicing/classes/view-models/invoicing-reservation-vm.ts
export interface InvoicingReservationVM {

    reservationId: string,
    destination: string,
    ship: string,
    port: string,
    ticketNo: string,
    adults: number,
    kids: number,
    free: number,
    totalPersons: number,
    embarkedPassengers: number,
    totalNoShow: number,
    remarks: string,
    hasTransfer: boolean

}

File: API/Features/Invoicing/ViewModels/InvoicingReservationVM.cs
using System;

namespace API.Features.Invoicing {

    public class InvoicingReservationVM {

        public string Date { get; set; }
        public string RefNo { get; set; }
        public Guid ReservationId { get; set; }
        public string Destination { get; set; }
        public string Port { get; set; }
        public string Ship { get; set; }
        public string TicketNo { get; set; }
        public int Adults { get; set; }
        public int Kids { get; set; }
        public int Free { get; set; }
        public int TotalPersons { get; set; }
        public int EmbarkedPassengers { get; set; }
        public int TotalNoShow { get; set; }
        public string Remarks { get; set; }
        public bool HasTransfer { get; set; }

    }

}